src/Raven.Server/Documents/Indexes/Static/DynamicArray.cs

```diff
@@ -335,6 +335,33 @@ public IEnumerable<dynamic> OrderByDescending(Func<IGrouping<dynamic, dynamic>,
             return new DynamicArray(_inner.Cast<DynamicGrouping>().OrderByDescending(comparable));
         }
 
+        public IOrderedEnumerable<object> CreateOrderedEnumerable<TKey>(Func<object, TKey> keySelector, IComparer<TKey> comparer, bool descending)
```

Comment:
This is not used?

Reply:
Implemented because of the inheritance from `IOrderedEnumerable`